### PR TITLE
Added Java EE example

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

**Comment by jpkrohling** [comment]
I created this example based on the `java-cdi` one, to test a report about running an OpenTracing application on WebSphere Liberty and decided to send this as a PR, as this might be useful to others.

File: examples/java-ee/pom.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        </dependency>
+    </dependencies>
+
+    <build>
+        <finalName>opentracing-jaxrs2-example-java-ee</finalName>
+        <plugins>
+            <plugin>
+                <groupId>org.wildfly.plugins</groupId>
+                <artifactId>wildfly-maven-plugin</artifactId>
+                <version>1.2.0.Alpha6</version>
```
Comment (pavolloffay) [inline examples/java-ee/pom.xml]: There is also 1.2.0.Final

**Comment by pavolloffay** [comment]
@jpkrohling sorry I totally forgot about this :/ 

Did it work on WebSphere Liberty?

**Comment by jpkrohling** [comment]
Yes, the readme has also instructions on running this example on Liberty.

**Comment by jpkrohling** [comment]
I'm closing this, as it's open since Oct without merging. If it's interesting to have this on this repo, feel free to reopen/merge.

**Pull request closed without merging**